src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -18,7 +17,7 @@ public static SnapshotBuilderConcept toConcept(FieldValueList row) {
     return new SnapshotBuilderConcept()
         .id((int) (row.get("concept_id").getLongValue()))
```

Comment:
can we make `concept_id` `public static final String`  like `has_children` and use that here? I think if we are going to use variables we should use them throughout?

Reply:
Sure, I can do that. I think we'll want to continue to refactor this code over time too.